### DISCUSSION TITLE
QR code with the latest release (March 28, 2023)

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2752] [Show and tell]
Can anyone who has updated to the latest release (March 28, 2023) try "Auto configure"?
Does the scanner come up for you?

**Comment by JavierDT1** [comment]
Tested in Samsung A7 (SM-A750FN/DS) with android 10... and scanner doesn´t open.

**Reply by Navid200** [reply]
Is it Android 10?

**Reply by JavierDT1** [reply]
Yes. Android 10

**Comment by Navid200** [comment]
| User | Phone | Android | Scanner |  
| ----- | ------- | --------- | ------------- |  
| Navid200 | Moto Z | 9 | Failed |  
| " | Moto Stylus | 11 | Failed |  
| JavierDT1 | Samsung | 10 | Failed |

**Comment by jamorham** [comment]
The nightly B version should fix the issue: https://github.com/NightscoutFoundation/xDrip/releases/download/2023.03.28b/xDrip-plus-20230328-940daa7.apk

**Reply by Navid200** [reply]
The b version works.  Thanks
